### DISCUSSION TITLE
Phone Numbers are not properly transcribed using smart format

### DISCUSSION THREAD

**Discussion by cateas** [deepgram/community#867] [General help] [answered]
Hi,

We are using smart format:

const deepgramLive: LiveClient = this.deepgram.listen.live({
  encoding: 'mulaw',
  endpointing,
  interim_results: true,
  language: 'en',
  model: 'nova-2',
  sample_rate: 8000,
  smart_format: true,
  utterance_end_ms: utterance,
  vad_events: true,
});

We noticed if  we provide a number like: 503-421-7400 (we are spelling it five oh three four two one seventy four hundred) as a result we get as transcription 503-421-74100 -- there is 1 added in front of 0.

This works properly with numerals, but we see it is marked as deprecated. Is it possible to use it with smart formatting ? 
Do you plan to remove the numerals in future? 

Thanks!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@cateas  Numerals isn't deprecated completely yet and is available for use as single query option. Smart Format does a lot of what that feature did, but if you are finding numerals working for you for phone numbers,  feel free to use it. 

Numerals will be available in our API until we decide to release a new API version. 

I'm curious where did you see it marked as deprecated?

**Comment by cateas** [comment]
Hi,

Good to know. Thanks.

From node_modules/@deepgram/sdk/src/lib/types/TranscriptionSchema.ts

/**
 * @see https://developers.deepgram.com/docs/numerals
 * @deprecated
 */
numerals?: boolean;
 
Thanks!

**Reply by jpvajda** [reply]
@cateas Thanks! Like I mentioned, the feature is considered `deprecated`, but it's still available to use for the forseeable future. If / When we do officially remove it from the API, we will certainly notify everyone as it be a major version bump of our API from V1 to V2.
